CHANGELOG.md

```diff
@@ -60,9 +60,9 @@ _None_
 
 ### Internal Changes
 
-* Integrated Danger to help on contributions by giving automatic feedback.  
+* Integrated Danger to help on contributions by giving automatic feedback.
   [Olivier Halligon](https://github.com/AliSoftware)
-  [#520](https://github.com/SwiftGen/SwiftGen/issues/520)
+  [#521](https://github.com/SwiftGen/SwiftGen/issues/520)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/SwiftGen/SwiftGen/blob/5cfc65c6621c24a0fbba81bd6db6799fde2356b4/CHANGELOG.md#L65"></span>link text is #521 but links points to SwiftGen/SwiftGen#520.</td>
    </tr>
  </tbody>
</table>